scripts/Diffraction/isis_powder/routines/focus.py

```diff
@@ -175,7 +175,6 @@ def _divide_by_vanadium_splines(spectra_list, vanadium_splines, instrument):
                        for data_ws, van_ws in zip(spectra_list, vanadium_splines)]
         return output_list
     output_list = [_divide_one_spectrum_by_spline(spectra_list[0], vanadium_splines, instrument)]
-    common.remove_intermediate_workspace(vanadium_splines)
```

Comment:
Can I just double check that the intermediate workspaces are now meant to be left afterwards and this isn't just for debugging?

Reply:
Yes this was intentional. For PEARL the vanadium splines get deleted by the call to DeleteWorkspace on ~L145 so the presence of the line we're talking about here caused an error because the workspace had already been deleted.

For instruments other than PEARL the situation before this PR was that the vanadium splines workspace was deleted if there were multiple banks but wasn't deleted if there was a single bank. So I've made this more consistent and for these instruments the vanadium workspace isn't deleted in all cases.